bin/stacks/routing-caching-stack.ts

```diff
@@ -37,6 +53,11 @@ export class RoutingCachingStack extends cdk.NestedStack {
       ],
     });
 
+    lambdaRole.addToPolicy(new PolicyStatement({
+      resources: ['*'], 
```

Comment:
Best to always scope down policies to have minimum positions. You can specify the role arn we're assuming here